app/Mail/FeedbackGiven.php

```diff
@@ -23,8 +23,9 @@ class FeedbackGiven extends Mailable
 
     public function __construct(Student $user, string $subject, string $description)
     {
+        $this->subject = null;
         $this->user = $user;
-        $this->subject = $subject;
+        $this->bug_subject = $subject;
```

Comment:
This should property should also be declared in the class as a string, like the `description` property.